src/cmds/qmgr.c

```diff
@@ -2211,25 +2211,29 @@ display(int otype, int ptype, char *oname, struct batch_status *status,
 				}
 			}
 			else  {
+				indent_len = 4;
 				if (otype == MGR_OBJ_RSC) {
 					if ((attr != NULL) && (strcmp(attr->name, "type") == 0)) {
 						struct resc_type_map *rtm = find_resc_type_map_by_typev(atoi(attr->value));
 						if (rtm) {
-							printf("\ttype = %s\n", rtm->rtm_rname);
+							printf("%*s", indent_len, " ");
+							printf("type = %s\n", rtm->rtm_rname);
```

Comment:
Did you have a reason to make these two printf()'s instead of one?  It's just one line of output